index.html

```diff
@@ -3,9 +3,27 @@
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="index.css">
+
     <title>postIt</title>
 </head>
 <body>
+<div class="wrapper">
+    <header>
+        <img class="logo" src="res/media/note.png" alt="logo">
+        <div class="search-container">
+            <form action="" >
+                <label>
+                    <input class="search-container" type="text" placeholder="">
+                </label>
+                <button type="submit">Search</button>
+            </form>
+        </div>
+
+        <img class="profile" src="res/media/profile.png" alt="profile">
+    </header>
+
+</div>
+
     <main>
         <div style="height: 100px"></div>
     </main>
```

Comment:
Remove wrapper, since header already is wrapper in this case. Change img classes to be more intuitive when referring from css. Remove div from main - It is not a good practice to write inline css. This div is also not necessary when correct height, width of elements is set in css.

    <header>
        <img class="img-logo" src="res/media/note.png" alt="logo">
        <div class="search-container">
            <form action="" >
                <label>
                    <input class="search-container" type="text" placeholder="">
                </label>
                <button type="submit">Search</button>
            </form>
        </div>
        <img class="img-profile" src="res/media/profile.png" alt="profile">
    </header>

    <main>
        <script type="module" src="res/js/index.js"></script>
    </main>